de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/invoicecandidates/impl/InvoicesRestControllerImpl.java

```diff
@@ -74,7 +84,7 @@ public InvoicesRestControllerImpl(
 			@ApiResponse(code = 403, message = "Accessing a related resource is forbidden"),
 			@ApiResponse(code = 422, message = "The request body could not be processed")
 	})
-	@PostMapping(path = "/createCandidates")
+	@PostMapping(path = "/createCandidates", consumes = "application/json", produces = "application/json")
```

Comment:
plase use either `value` or `path` in all this controller's methods. After a bit of research, i think `path` would be clearer than `value`, but i don't have strong feelings in favor for or against either..

Reply:
done. Used path everywhere